firmware/common2015/drivers/ball_sense/BallSense.hpp

```diff
@@ -36,7 +39,7 @@ class BallSense {
 
     // If the light reading and dark reading are closer than this value, the
     // beam is considered broken
-    const int sense_threshold = 5;
+    const int sense_threshold = 500;
```

Comment:
Me and Justin tuned it in the shop and it seemed pretty good, we tested it in both dark and light conditions.